### PR TITLE
Header section

### PR DESCRIPTION
**Pull request by sunmadetek**
The first pull request for the first project

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <main>
+        <div id="navdiv">
+        <ul>
+        <li ><a href="#Olasupo Ismail">Olasupo Ismail</a></li>
+        <li id="shift"><a href="#"><i class="fa-solid fa-envelope"></i></a></li>
+        <li id="shift"><a href="#Portfolio">Portfolio</a></li>
```
Comment (brainconnect93) [inline index.html]: the list tag in line 14 should be an header either h1-h6

File: index.html [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+        <li id="shift"><a href="#About">About</a></li>
+        <li id="shift"><a href="#Contact">Contact</a></li>
+        </ul>
```
Comment (brainconnect93) [inline index.html]: kindly add the message icon here

File: index.html [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+            </tr>
+            <tr>
+            <td>Ruby</td>
+            <td>Ruby on Rails</td>
+            <td>Github</td>
+            </tr>
+            <tr>
+            <td>HTML</td>
+            <td>RSPec</td>
+            <td>Codepen</td>
```
Comment (brainconnect93) [inline index.html]: it's nice using the table HTML semantic but I would suggest you have this section writing inform of a list tag

**Review by brainconnect93 — CHANGES_REQUESTED** [review]
Hi @Sunmade1 👏 
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
image

## REQUIRED CHANGES

File: index.html (hunk 1 above)
Comment (sunmadetek) [reply index.html]: You header element nested in the list  tag?

**Pull request merged**